mtools.py

```diff
@@ -24,6 +39,11 @@ def sd(input_list):
     n = len(input_list)
     return (s / (n - 1)) ** 0.5
 
+if __name__ == '__main__':
```

Comment:
```suggestion

```

Reply:
I would remove this debugging test from here, maybe leave something for examples if run as script at the end.